casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -731,7 +731,7 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
       .ensureOr(
         validatorsPar =>
           new IllegalArgumentException(
-            s"Incorrect number of results from query of current active balidator: ${validatorsPar.size}"
+            s"Incorrect number of results from query of current active validator in state ${startHash}: ${validatorsPar.size}"
```

Comment:
Do we need `PrettyPrinter.buildString` for `startHash`? (the same in computeBonds)

Reply:
Yes, you are right!